### ISSUE TITLE
Binary data handling and especially NUL bytes

### ISSUE THREAD

**Issue by wagner-certat** [certtools/intelmq#967]
Shadowserver feeds can contain binary data including NULL bytes which results in errors when the csv parser tries to read these lines:
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.4/csv.py", line 110, in __next__
    row = next(self.reader)
_csv.Error: line contains NULL byte
```
Further lines can't be parsed.

We ask upstream Shadowserver for a solution and an intermediate solution would be the removal of all NULL bytes before parsing.

**Comment by wagner-certat** [comment]
Shadowserver confirmed the issue and is working on it.

However I discover that the problem in IntelMQ lies deeper than expected. When inserting NULL bytes in testdata, everything works fine.

**Comment by wagner-certat** [comment]
I pushed an intermediate quick fix to master: b099482

**Comment by wagner-certat** [comment]
Moving to 1.1 as we have an fix which is good enough for 1.0

**Comment by dmth** [comment]
We've backported the fix to our integrated branch.
